Wikipedia.xcodeproj/xcshareddata/xcschemes/Wikipedia.xcscheme

```diff
@@ -86,6 +87,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      disableMainThreadChecker = "YES"
```

Comment:
Can this be removed and only have the change on the "Test" config?

Reply:
I'll try...

Reply:
Fixed here: https://github.com/wikimedia/wikipedia-ios/pull/2314/commits/1238c8cd7e13e6f4a12d71c94f17e79dcf8ede59